.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1 +1,2 @@
-distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
\ No newline at end of file
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
+#wrapperUrl=http://central.maven.org/maven2/io/takari/maven-wrapper/0.2.1/maven-wrapper-0.2.1.jar
```

Comment:
Should be the https URL and also be the same repo1.maven.org as above

Reply:
Also should this really be commented out?

Reply:
Well it is in there as an example of how to configure the wrapper. It is equal to the default used in the wrapper and downloader code. Sort of wanted to imply that you don't need that property, but you could use it and how you would have to use it.